langchain/llms/llamacpp.py

```diff
@@ -64,6 +64,9 @@ class LlamaCpp(LLM):
     """Number of tokens to process in parallel.
     Should be a number between 1 and n_ctx."""
 
+    n_gpu_layers: Optional[int] = 0
```

Comment:
as far as i can tell this isn't optional in llama_cpp package, so probably don't want to allow it to be here?

Reply:
same as other params in this wrapper, in llama-cpp-python progect it's optional with default value 0
https://github.com/abetlen/llama-cpp-python/blob/main/llama_cpp/llama.py#L86

tested without this param:
`llm = LlamaCpp(model_path=...)`

Output:
`llama_model_load_internal: [cublas] offloading 0 layers to GPU`

Reply:
yes, correct in that regards, updated, is it better now
`n_gpu_layers: Optional[int] = Field(0, alias="n_gpu_layers")`

Reply:
im suggesting we make it `n_gpu_layers: int = ...`

Reply:
actually updated to leave it as None and not pass in if it is None, since that would break backwards compatability ( (since n_gpu_layers support was only recently added to llama-cpp-python)